code/modules/martial_arts/martial.dm

```diff
@@ -271,6 +271,35 @@
 	new /obj/effect/decal/cleanable/ash(get_turf(src))
 	qdel(src)
 
+/obj/item/tae_clown_do_scroll
+	name = "scroll of tae clown do"
+	desc = "A scroll filled with hilarious markings. It seems to be drawings of some sort of silly martial art."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "clownscroll"
+
+/obj/item/tae_clown_do_scroll/attack_self(mob/living/carbon/human/user)
+	if(!istype(user))
+		return
+	//Prevents changelings and vampires from being able to learn it
+	if(user.mind?.changeling) //Changelings
+		to_chat(user, "<span class='warning'>We try multiple times, but we are not unable to focus on the contents of the scroll as we continue bursting out in laughter!</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>We try multiple times, but are unable to focus on the contents of the scroll because we repeatedly burst out in laughter!</span>")
```